src/pipeline/okta_ingestion.py

```diff
@@ -0,0 +1,62 @@
+import alooma_pysdk
```

Comment:
duplicate file?

Reply:
Ah, this was a casualty of the rename from pipeline to ingestion. Should be removed now.